ROMFS/px4fmu_common/init.d/rc.uuv_defaults

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# UUV default parameters.
+#
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
+
+set VEHICLE_TYPE uuv
+
+if [ $AUTOCNF = yes ]
+then
+        param set PWM_MAX 1950
```

Comment:
I don't think these PWM params make sense as UUV defaults.

Reply:
Well, I agree with you on min/max - but doesn't it make sense to have the disarmed pwm at 1500? Better ways of doing this are welcomed =)